src/components/emby-select/emby-select.js

```diff
@@ -116,7 +116,7 @@ define(['layoutManager', 'browser', 'actionsheet', 'css!./emby-select', 'registe
             inputId++;
         }
 
-        if (!browser.firefox) {
+        if (browser) {
```

Comment:
this conditional is useless now so it should be removed

Reply:
@dkanada If i remove the codition some part doesn't work that look like this
![Screenshot_2019-06-01 Jellyfin(2)](https://user-images.githubusercontent.com/32230989/58752714-697c9080-84bc-11e9-88ec-b4ecd9eb6757.png)